src/Microsoft.PowerApps.TestEngine/Modules/TestEngineExtensionChecker.cs

```diff
@@ -243,16 +247,21 @@ public virtual bool Validate(TestSettingExtensions settings, string file)
         {
             var contents = GetExtentionContents(file);
```

Comment:
Define explicitly the type here (it is not possible for a reviewer to infer the type) and should we move this line next to 265 closer when is the first use of this var?

Reply:
added and moved